src/components/table/JournalEntryTable.tsx

```diff
@@ -0,0 +1,121 @@
+import { CustomTheme } from '@mui/material/styles';
+
+import LightTable from './LightTable';
+
+const styles = (theme: CustomTheme) => ({
+  textWrapper: {
+    ...theme.mixins.ellipsis,
+  },
+  statusContainer: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  statusIcon: {
+    fontSize: '1rem',
+    marginRight: 10,
+  },
+  statusIconSuccess: {
+    color: theme.palette.success.light,
+  },
+  statusIconFailed: {
+    color: theme.palette.error.light,
+  },
+});
+
+interface IProps extends WithStyles<typeof styles> {
+  journalEntries: Array<JournalEntryModel>;
+  journalEntriesCount?: number;
+  filteredJournalEntriesCount?: number;
+  handleFilterJournalEntries?: (
+    rowsPerPage: number,
+    rowsForCurrentPage: number,
+    searchString: string,
+  ) => void;
+  loading: boolean;
+  refreshCnt: number;
+}
+
+const JournalEntryTable = withStyles(styles)(
+  ({ classes, journalEntries, handleFilterJournalEntries, loading, refreshCnt }: IProps) => {
+    const statusMap: any = {
+      CREATE: { icon: `fa fa-circle-check ${classes.statusIconSuccess}`, label: 'Created' },
+      UPDATE: { icon: `fa fa-circle-check ${classes.statusIconSuccess}`, label: 'Updated' },
+    };
+
+    const getStatusLabel = (row: JournalEntryModel) => {
+      if (row.entryType === 'CREATE') {
+        return 'Created';
+      }
+      if (row.entryType === 'UPDATE' && row.methodName === 'lambda$addPolicyMember$15') {
+        return 'Shared';
+      }
+      if (row.entryType === 'UPDATE' && row.note?.includes('Ingested')) {
+        return 'Ingested';
+      }
+      return row.entryType;
+    };
+
+    const columns: Array<TableColumnType> = [
+      {
+        label: 'Initiator',
+        name: 'user',
+        allowSort: false,
+        width: '20%',
+        cellStyles: {
+          whiteSpace: 'normal',
+        },
+      },
+      {
+        label: 'Action',
+        name: 'note',
+        allowSort: false,
+        width: '20%',
+        cellStyles: {
+          whiteSpace: 'normal',
+        },
+      },
+      {
+        label: 'Date',
+        name: 'when',
+        allowSort: false,
+        render: (row: any) => moment(row.when).fromNow(),
+        width: '10%',
+      },
+      {
+        label: 'Status',
+        name: 'entryType',
+        allowSort: false,
+        width: '10%',
+        render: (row: any) => (
+          <span className={classes.statusContainer}>
+            <i className={clsx(classes.statusIcon, statusMap[row.entryType].icon)} />
+            {getStatusLabel(row)}
+          </span>
+        ),
+      },
+    ];
+    return (
+      <LightTable
+        columns={columns}
+        handleEnumeration={handleFilterJournalEntries}
+        noRowsMessage="No journal entries found"
+        infinitePaging={true}
+        filteredCount={Number.MAX_SAFE_INTEGER}
+        rows={journalEntries}
+        searchString=""
+        loading={loading}
+        refreshCnt={refreshCnt}
+        rowKey="id"
```

Comment:
I'm seeing some wonkiness with the paging of the journal table. I think we usually set the[ totalCount field](https://github.com/DataBiosphere/jade-data-repo-ui/blob/develop/src/components/table/DatasetTable.tsx#L112).
https://user-images.githubusercontent.com/13254229/201801625-5c81be25-b1a9-4cbd-91be-ff322f2ae45a.mov

Reply:
Oh, it looks like that is happening with the infinitePaging option. Is this behaving as expected?

Reply:
Unfortunately, that is expected behavior - It's because these requests are not returning any paging information whatsoever, so it's impossible to determine from the ui when to disable paging.

I think this weird behavior was less visible on the jobs page because there are a ton of jobs! One change we could make is to disable the next button if there are no results in the current view (or < pageSize results)... Let me try to push that into this PR

Reply:
Ah, true. Yeah, if you could add that, I think that would be great.